Runtime/Scripts/MultiAppManager.cs

```diff
@@ -39,26 +45,37 @@ public static string GetModelLoader()
 		return MAMGetModelLoader();
 	}
 
-	public static void LaunchApp(string appUrl, Vector3 position, Quaternion rotation, Vector3 scale)
+	public static void LaunchApp(string appUrl, Vector3 position, Quaternion quaternion, Vector3 scale)
 	{
-		MAMLaunchApp(appUrl, transformJSONFromElements(position, rotation, scale));
+		MAMLaunchApp(appUrl, transformJSONFromElements(position, quaternion, scale));
 	}
 
 	public static void GetApps(string receiverName, string dataMethodName, string errorMethodName)
 	{
 		MAMGetApps(receiverName, dataMethodName, errorMethodName);
 	}
 
-	public static void LaunchAppByNameId(string name, string id, Vector3 position, Quaternion rotation, Vector3 scale)
+	public static void GetAssets(string receiverName, string dataMethodName, string errorMethodName)
 	{
-		MAMLaunchAppByNameId(name, id, transformJSONFromElements(position, rotation, scale));
+		MAMGetAssets(receiverName, dataMethodName, errorMethodName);
 	}
 
-	private static string transformJSONFromElements(Vector3 position, Quaternion rotation, Vector3 scale)
+	public static void LaunchAppByNameId(string name, string id, Vector3 position, Quaternion quaternion, Vector3 scale)
+	{
+		MAMLaunchAppByNameId(name, id, transformJSONFromElements(position, quaternion, scale));
+	}
+
+	public static void LaunchAssetByNameId(string name, string id, Vector3 position, Quaternion quaternion, Vector3 scale)
+	{
+		MAMLaunchAssetByNameId(name, id, transformJSONFromElements(position, quaternion, scale));
+	}
+
+
+	private static string transformJSONFromElements(Vector3 position, Quaternion quaternion, Vector3 scale)
 	{
         return @$"{{
```

Comment:
oof! Escape chars in WASM?

Reply:
No WASM connection here that I know of: I've got the @ to let me do a multiline and the $ to let me do interpolation, but since `{}` are the interpolation delimiters, I have to do `{{` and `}}` to escape literal curlies for the JSON.